src/components/EnterBarcode/EnterBarcodePage.tsx

```diff
@@ -67,7 +67,11 @@ const EnterBarcodePage = ({
         </Button>
       </Box>
       <Box padding="56px">
-        <form>
+        <form
+          onSubmit={(e) =>
+            barcode !== '' ? handleEnterBarcode() : e.preventDefault()
```

Comment:
Is this alright or should it be put in a function and called onSubmit instead?
```
const handleFormSubmission = (e) => {
     barcode !== '' ? handleEnterBarcode() : e.preventDefault()
}

...

<form onSubmit={(e) => handleFormSubmission(e)}>
```

Reply:
i think this is just a matter of preference, up to you if you want to make the change - it can be cleaner and easier to edit in the future

also small nit for your example - `<form onSubmit={handleFormSubmit}>` should work as well - if you are gonna make the change i would test it first though

Reply:
Ok I've updated it - this ticket should be complete 😅